ui/app/styles/adt/_adt.scss

```diff
@@ -83,9 +83,18 @@ div[ng-app="adt"] {
         max-width: $max-page-width;
         overflow-x: auto;
 
+        input {
```

Comment:
A slight adjustment to the input box SCSS could align it with the appearance of our other text input boxes. Personally, I would prefer consistency in the look of the input box, matching it with others.

cc: @gokultw, @angshu, @binduak 

Current Input Box           |  Input Box In Other Pages |  Updated Input Box
:-------------------------:|:-------------------------:|:-------------------------:
![Screenshot 2024-04-10 at 4 38 19 PM](https://github.com/Bahmni/openmrs-module-bahmniapps/assets/121226043/c3d00c46-156c-491e-bd7e-b5457046bdd9)  | ![Screenshot 2024-04-10 at 4 51 08 PM](https://github.com/Bahmni/openmrs-module-bahmniapps/assets/121226043/2af68476-58af-4210-a788-686ef07b1270) | ![Screenshot 2024-04-10 at 4 50 47 PM](https://github.com/Bahmni/openmrs-module-bahmniapps/assets/121226043/1a53730c-5dbc-49b6-8367-45bc13b237b8)

```
input {
            float: left;
            min-width: 300px;
            margin: 10px 10px;
            padding: 2px 10px;
            &:focus {
                outline: 2px solid #007fff;
			}
            border: 1px solid #DDD;
        }
```

Reply:
@rahu1ramesh let me check with Adi on the design of search box

Reply:
@binduak Following today's discussion, as decided we will proceed with placing the search input box on the left side of the screen, consistent with other screens. The required adjustments have already been implemented to accommodate this change.